src/main/java/frc/robot/OI.java

```diff
@@ -8,13 +8,18 @@
 public class OI {
     //Operator Interface (OI) class containing all control information
     private static final int kDriverJoystickPort = 0;
+    private static final int kTeleopJoystickPort = 1;
 
     public static final class Driver{
         private static final Joystick kJoystick = new Joystick(OI.kDriverJoystickPort);
+        private static final Joystick kTeleopJoystick = new Joystick(OI.kTeleopJoystickPort);
 
         private static final int kOrientationButtonID = 1; //1 Button, Toggle swerve orientation
         private static final int kZeroButtonID = 2; //2 Button, Zero the gyroscope
 
+        private static final int kLedToggleID = 1; //Toggles Leds
+        private static final int kLedSwtichID = 2;
+
```

Comment:
Add comments describing what controller buttons these are supposed to be bound to (Ex. A Button, B Button, Right Trigger), what they're supposed to do (Toggle LED doesn't mean much - Toggles them on/off? Toggles them yellow/purple?)

Reply:
Got it!